src/components/ApiHistoryTable.tsx

```diff
@@ -0,0 +1,160 @@
+import { Details } from '@docusaurus/theme-common/Details';
+import React from 'react';
+import ReactMarkdown from 'react-markdown';
+
+import {
+  ApiHistory,
+  PrReleaseVersions,
+  PrReleaseVersionsContainer,
+} from '../transformers/api-history';
+import styles from './ApiHistoryTable.module.scss';
+
+enum Change {
+  ADDED = 'API ADDED',
+  CHANGED = 'API CHANGED',
+  DEPRECATED = 'API DEPRECATED',
+}
+
+interface ApiHistoryTableProps {
+  apiHistoryJson: string;
+  prReleaseVersionsJson: string;
+}
+
+function generateTableRow(
+  prReleaseVersions: PrReleaseVersions | undefined,
+  type: Change,
+  change: NonNullable<ApiHistory[keyof ApiHistory]>[0]
+) {
+  const prNumber = Number(change['pr-url'].split('/').at(-1));
+  const key = prNumber + '-' + type;
```

Comment:
```suggestion
  const key = change['pr-url'] + '-' + type;
```

Unless keys can't have special characters like slashes or colons, seems unnecessary to extract the PR number.